src/proxy.sol

```diff
@@ -50,19 +50,26 @@ contract DSProxy is DSAuth, DSNote {
         response = execute(target, _data);
     }
 
+
     function execute(address _target, bytes _data)
         public
         auth
         note
         payable
-        returns (bytes32 response)
+        returns (bytes response)
     {
         require(_target != 0x0);
 
         // call contract in current context
         assembly {
-            let succeeded := delegatecall(sub(gas, 5000), _target, add(_data, 0x20), mload(_data), 0, 32)
-            response := mload(0)      // load delegatecall output
+            let succeeded := delegatecall(sub(gas, 5000), _target, add(_data, 0x20), mload(_data), 0, 0)
```

Comment:
@NiklasKunkel can you explain the original choice of 5000 here? I wonder if there is a way to make this more dynamic, in case of future CALL pricing changes,

Reply:
At a minimum this needs to be 703 for the base call cost and one push/sub (not sure exactly where the 3 is from).

Reply:
Should we do this value dynamic as proposed in another PR? Do we leave fixed 5K? or change to another number?